EIPS/eip-5606.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: 5606
+title: Proxy NFTs
```

Comment:
Maybe add something differentiating what kind of proxy this is (like a voting proxy, or a software proxy, etc.)